UniversalAdapter/src/main/java/com/raizlabs/universaladapter/converter/MergedUniversalAdapter.java

```diff
@@ -183,14 +187,23 @@ public ListPiece getPieceAt(int position) {
 
     /**
      * @param adapterIndex The index of adapters added to this adapter. 0 is for the first adapter added, 1 is for second. etc.
-     * @param <Item>       The item of the adapter.
-     * @param <Holder>     The holder it uses.
      * @return The specified adapter from the adapterIndex.
      */
     public UniversalAdapter getAdapter(int adapterIndex) {
         return listPieces.get(adapterIndex).adapter;
     }
 
+    /**
+     * When data changes we need to update the starting positions of all adapters.
+     */
+    private void recalculateStartPositions() {
+        int previousCount = 0;
```

Comment:
#nth This name sounds like "the count before whatever just happened". Maybe something like `lastIndex` or something would be more clear?